lib/pymedphys/docs/_config.yml

```diff
@@ -4,17 +4,19 @@ copyright: "2018-2021"
 logo: img/pymedphys.svg
 execute:
   execute_notebooks: cache
-parse:
-  myst_extended_syntax: true
+# parse:
```

Comment:
why did this have to be commented out?  Was this part of the 0.41 release and already discussed?

Reply:
Sorry, I think I was trying to resolve some of the documentation warnings that sphinx throws when building. You know, for funsies. But didn't really get anywhere, and really should've been working on a different branch. I've reverted those changes.